charts/ocis/templates/appprovider/deployment.yaml

```
@@ -69,7 +71,7 @@ spec:
               value: {{ tpl $.Values.features.appsIntegration.wopiIntegration.wopiFolderURI $ | quote }}
 
             - name: OCIS_REVA_GATEWAY
-              value: {{ $.appNameGateway }}:9142
+              value: com.owncloud.api.gateway
```

Can this magic string change? At least the appName part? If yes it might be changed into

```suggestion
              value: com.owncloud.api.{{ $.appNameGateway }}
```

Same question applies to all appearances below.